how-to/misc/adding-amd-sev-enabled-compute-node.rst

```diff
@@ -0,0 +1,59 @@
+Adding AMD SEV enabled Compute Node
+===================================
+
+Secure Encrypted Virtualization is a technology from AMD that enable encryption of guest memory.
+OpenStack provides support to make use of this technology and deploy trusted guests.
+Refer `Admin guide <https://docs.openstack.org/nova/latest/admin/sev.html>`_ for more information.
+
+In Canonical OpenStack, AMD SEV enabled compute nodes can be added using the :doc:`cluster scale out procedure </how-to/operations/scaling-the-cluster-out>` by adding compute role to the node.
+Canonical OpenStack auto detects the compute node if the node is AMD SEV enabled or not.
+
+For AMD SEV enabled compute nodes, sufficient memory need to be reserved for the host since SEV enabled guests memory pages are pinned in RAM.
+To set the reserved memory for the host, update manifest with the following configuration for openstack-hypervisor charm.
```

Comment:
This bit about the manifest is not clear.

How do you apply that? Do we want people to do that only when joining a new node?

Reply:
Commands added for join/deploy